process.go

```diff
@@ -532,54 +532,65 @@ func ProcessFootageItem(client *mongo.Client, adminSetting Adminsetting, item It
 	if err != nil {
 		return err
 	}
-	/*
-		// .mov 썸네일 동영상을 생성한다.
-		err = SetStatus(client, item, "creatingmovmedia")
-		if err != nil {
-			return err
-		}
-		err = genProxyToMovMedia(adminSetting, item) // FFmpeg는 확장자에 따라 옵션이 다양하거나 호환되지 않는다. 포멧별로 분리한다.
-		if err != nil {
-			// 상태를 error로 바꾼다.
-			err = SetStatus(client, item, "error")
-			if err != nil {
-				return err
-			}
-			// 에러 내용을 로그로 남긴다.
-			err = SetLog(client, item.ID.Hex(), err.Error())
-			if err != nil {
-				return err
-			}
-			return err
-		}
-		err = SetStatus(client, item, "createdmovmedia")
+
+	// .mov 썸네일 동영상을 생성한다.
+	err = SetStatus(client, item, "creatingmovmedia")
+	if err != nil {
+		return err
+	}
+	err = genProxyToMovMedia(adminSetting, item) // FFmpeg는 확장자에 따라 옵션이 다양하거나 호환되지 않는다. 포멧별로 분리한다.
+	if err != nil {
+		// 상태를 error로 바꾼다.
+		err = SetStatus(client, item, "error")
 		if err != nil {
 			return err
 		}
-		// .mp4 썸네일 동영상을 생성한다.
-		err = SetStatus(client, item, "creatingmp4media")
+		// 에러 내용을 로그로 남긴다.
+		err = SetLog(client, item.ID.Hex(), err.Error())
 		if err != nil {
 			return err
 		}
-		err = genProxyToMp4Media(adminSetting, item) // FFmpeg는 확장자에 따라 옵션이 다양하거나 호환되지 않는다. 포멧별로 분리한다.
+		return err
+	}
+	err = SetStatus(client, item, "createdmovmedia")
+	if err != nil {
+		return err
+	}
+	// .mp4 썸네일 동영상을 생성한다.
+	err = SetStatus(client, item, "creatingmp4media")
+	if err != nil {
+		return err
+	}
+	err = genProxyToMp4Media(adminSetting, item) // FFmpeg는 확장자에 따라 옵션이 다양하거나 호환되지 않는다. 포멧별로 분리한다.
+	if err != nil {
+		// 상태를 error로 바꾼다.
+		err = SetStatus(client, item, "error")
 		if err != nil {
-			// 상태를 error로 바꾼다.
-			err = SetStatus(client, item, "error")
-			if err != nil {
-				return err
-			}
-			// 에러 내용을 로그로 남긴다.
-			err = SetLog(client, item.ID.Hex(), err.Error())
-			if err != nil {
-				return err
-			}
 			return err
 		}
-		err = SetStatus(client, item, "createdmp4media")
+		// 에러 내용을 로그로 남긴다.
+		err = SetLog(client, item.ID.Hex(), err.Error())
 		if err != nil {
 			return err
 		}
-	*/
+		return err
+	}
+	err = SetStatus(client, item, "createdmp4media")
+	if err != nil {
+		return err
+	}
+
+	// Proxy 제거
+	err = os.RemoveAll(item.OutputProxyImgPath)
+	if err != nil {
+		return err
+	}
+	err = SetStatus(client, item, "remove proxy")
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	err = SetStatus(client, item, "removing proxy")
	if err != nil {
		return err
	}
```
위 코드를 os.RemoveAll 전에 두는 건 어때요?